110-balanced-binary-tree.py

```diff
@@ -0,0 +1,70 @@
+
+"""
+110. Balanced Binary Tree
+Easy
+"""
+from leetcode_runner import LeetCode, TestCase, Args
+from typing import *
+
+PROBLEM = """
+Given a binary tree, determine if it is height-balanced.
+ 
+Example 1:
+
+
+Input: root = [3,9,20,null,null,15,7]
+Output: true
+
+Example 2:
+
+
+Input: root = [1,2,2,3,3,null,null,4,4]
+Output: false
+
+Example 3:
+
+Input: root = []
+Output: true
+
+ 
+Constraints:
+
+The number of nodes in the tree is in the range [0, 5000].
+-104 <= Node.val <= 104
+
+
+"""
+
+# Definition for a binary tree node.
+# class TreeNode:
+#     def __init__(self, val=0, left=None, right=None):
+#         self.val = val
+#         self.left = left
+#         self.right = right
+class Solution:
+    """
+    Accepted
+    Time Complexity: O(n^2)
```

Comment:
Submit the optimized solution as well for this problem

Reply:
Its in the second commit.
Please let me know if uts not in the PR